components/lib/TabsHeader/TabsHeader.module.scss

```diff
@@ -26,7 +25,6 @@
   bottom: -1px;
   left: 50%;
   transform: translateX(-50%);
-  width: 2px;
```

Comment:
It will remove animation of tab bottom line, please resolve it without removing animation of bottom line